### ISSUE TITLE
Stuck Bounties

### ISSUE THREAD

**Issue by Wolfspirit445** [Rocologo/MobHunting#103]
V4.1.2

Bounties still exist in "/mh bounty" (both GUI and noGUI) after removing a player bounty. MH recognizes that a bounty does not exist for a player who had their bounty removed, but this bounty still appears with the "/mh bounty" command.

The command "mh bounty [player] drop" does not appear to work. Instead, "/mh bounty remove [player]" works. (Neither removes a player bounty from "/mh bounty", but "/mh bounty remove [player]" does say a player bounty was removed and I received 50% of the $ back for the bounty.
![2016-11-27_15 22 05](https://cloud.githubusercontent.com/assets/24213254/20652401/c4ed5e24-b4b5-11e6-8093-e9e56dcd4839.png)
![2016-11-27_15 22 24](https://cloud.githubusercontent.com/assets/24213254/20652405/caa25464-b4b5-11e6-85c0-91dd88edb2c8.png)
![2016-11-27_15 23 02](https://cloud.githubusercontent.com/assets/24213254/20652406/d2e61b42-b4b5-11e6-95c9-1907b4a91a02.png)

**Comment by Rocologo** [comment]
You can now use both "drop" and "remove" and I hope I was able to move the bounty from from the /mh bounty list. I need your help to help me testing if it works now, because its hard to test on my test server with only me as a player :-)

Please try this dev build: 
https://dl.dropboxusercontent.com/u/36067670/MobHunting/MobHunting-4.1.3-SNAPSHOT-20161128122022-B217.jar

**Comment by Wolfspirit445** [comment]
I had to delete the original database and put a bounty on a player again to test. It looks like I am now able to correctly remove a bounty on a player with "/mh bounty remove [player]", but I am still unable to use the original "/mh bounty [player] drop". Using this just brings up the list "usage" for the /mh bounty commands.

Thank you, very much for the speedy response and build. I switched over from MoneyMobs and this plugin is fantastic!
